### PR TITLE
FIX: Add DWI to the default queries of BIDS querying

### PR DESCRIPTION
**Pull request by oesteban**
This PR adds DWI to the queriable suffixes, and also opens a possibility to modify the default queries.

### CODE REVIEW

File: niworkflows/utils/bids.py [hunk 1]
```diff
@@ -223,19 +237,10 @@ def collect_data(
         'return_type': 'file',
         'subject': participant_label,
         'extension': ['.nii', '.nii.gz'],
-        'session': session_id,
+        'session': session_id or Query.OPTIONAL,
```
Comment (effigies) [inline niworkflows/utils/bids.py:240]: This changes the meaning of None from Query.NONE, which diverges from pybids semantics. I think that's an okay change, but we should check that we never use None in the way now.

Comment (oesteban) [reply niworkflows/utils/bids.py:241]: You are right, didn't think of it. Considering that single session and multisession datasets cannot be mixed up in BIDS, I agree this should be okay.

File: niworkflows/utils/bids.py [hunk 2]
```diff
@@ -151,11 +164,13 @@ def collect_participants(
 def collect_data(
     bids_dir,
     participant_label,
-    session_id=Query.OPTIONAL,
+    session_id=None,
     task=None,
     echo=None,
+    group_me=True,
```
Comment (effigies) [inline niworkflows/utils/bids.py:170]: Non-obvious parameter name. And please add an entry to the docstring.

```suggestion
    group_multiecho=True,
```

**Review by effigies — COMMENTED** [review]
I've got I think all of the major nipreps checked out in one directory, and grepping for uses that include `session_id`, we only get nibabies. That one looks safe.

One small comment and this is good to go.

File: niworkflows/utils/bids.py (hunk 2 above)
Comment (oesteban) [reply niworkflows/utils/bids.py:170]: I believe I did that because the name overloads the function defined below, but I can modify the function name.

Comment (effigies) [reply niworkflows/utils/bids.py:170]: Or `group_echos`?

Comment (effigies) [reply niworkflows/utils/bids.py:170]:
```suggestion
    group_echos=True,
```

File: niworkflows/utils/bids.py [hunk 3]
```diff
@@ -256,7 +262,11 @@ def collect_data(
     }
 
     # Special case: multi-echo BOLD, grouping echos
-    if any(["_echo-" in bold for bold in subj_data["bold"]]):
+    if (
+        group_me
```
Comment (oesteban) [inline niworkflows/utils/bids.py:266]:
```suggestion
        group_echos
```

**Pull request merged**